aten/src/ATen/native/vulkan/VulkanOps.cpp

```diff
@@ -473,8 +490,10 @@ ImageSizes conv2d_prepack_weights_image_sizes(
     int64_t C,
     int64_t KH,
     int64_t KW) {
-  return {{ALIGN_UP4(C), UP_DIV(OC, 4), KH * KW},
-          {ALIGN_UP4(C), UP_DIV(OC, 4), KH * KW}};
+  const uint32_t Cup4 = ALIGN_UP4(static_cast<uint32_t>(C));
```

Comment:
I should have mentioned this earlier, but inline functions (and functions in general) are superior to macros.  A change you can consider in a later PR.